client/app/telecom/pack/migration/building-details/telecom-pack-migration-building-details.controller.js

```diff
@@ -70,9 +77,14 @@ angular.module('managerApp').controller('TelecomPackMigrationBuildingDetailsCtrl
   }
 
   isValidSelection() {
-    if (this.model.selectedBuilding != null
-      && this.model.selectedStair != null && this.model.selectedFloor != null) {
-      return true;
+    if (this.model.selectedBuilding != null && this.model.selectedStair != null
+      && this.model.selectedFloor != null && this.model.pto != null) {
+      if (this.model.pto && ((this.model.ptoReference != null && this.model.ptoReference !== '') || this.model.ptoReferenceNotKnown)) {
+        return true;
+      } if (!this.model.pto) {
```

Comment:
Can `this.model.pto` have a falsy value apart from `null` ? If not this will never be executed

Reply:
yes it has. this.model.pto is linked to the radio button (true or false values)